### PR TITLE
{bio} [goolf-1.4.10] INRICH-1.1 (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

File: easybuild/easyconfigs/i/INRICH/INRICH-1.1-goolf-1.4.10_30c7f7e.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'MakeCp'
+
+name = 'INRICH'
+version = '1.1'
+git_commit = '30c7f7e' # no direct download url so we download the git commit of latest release
+versionsuffix = '_%s' % git_commit
```
Comment (boegel) [inline easybuild/easyconfigs/i/INRICH/INRICH-1.1-goolf-1.4.10_30c7f7e.eb]: @pescobar use `-` rather than `_`?

**Comment by boegel** [comment]
@pescobar missing dep on SQLite:

```
g++ -O2 -c loaders.cpp
7080 In file included from loaders.cpp:2:0:
7081 lddb.h:4:21: fatal error: sqlite3.h: No such file or directory
7082 compilation terminated.
7083 make: *** [loaders.o] Error 1
```

Also, not picking up `$CFLAGS`?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
gligar03.gligar.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a74a454702f1a22c9d20deac3275fa0c for a full test report.

**Comment by boegel** [comment]
@pescobar ping?

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**